inloop/statistics/validators.py

```diff
@@ -33,7 +42,7 @@ def get_optional_bool(key_name: str, data: dict) -> Optional[bool]:
     Get and validate a boolean parameter from a data dictionary.
     """
     parameter = data.get(key_name)
-    if parameter is None:
+    if parameter is None or parameter == "":
```

Comment:
As you can see, we try to unwrap a boolean here, for example from a string or from an already unwrapped boolean itself. Committing your suggestion would lead to `False` (in its already unwrapped form) being parsed as `None`.

Reply:
Note, that these validators should be handled as if they weren't only combined with string values, but also the already unwrapped values themselves, such as in
```python
get_optional_bool('my_bool', {'my_bool': False})
```

Reply:
Hm, now I am beginning to question the exact purpose of these functions ;-) I thought you use them with `request.GET` dictionary, which maps strings to strings. Why should they be able to handle already unwrapped values? Or aren't you sure if the values (if any) of the `GET` dict are always strings?

Reply:
As of now, the purpose of these functions is to safely unwrap a value from a dictionary, where this value can be given in any data format nested inside the dictionary. I designed it like this to be more reusable in other use cases where it is not guaranteed, that the data format of the nested value is `str`.

Reply:
Okay. Just saying, keep in mind the YAGNI (you ain't gonna need it) principle. Furthermore, if it should be that general it should also be able to convert '1' and '0' to True and False, respectively. I just was commenting about that because the function seemed overly complex and I was wondering if that's really necessary. Anyway, I'll accept it like that and see myself what I can do to shorten it.